LICENCE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2019-present NEM
+   (C) Copyright Symbol Contributors 2021
```

Comment:
The format is [Copyright Symbol] [Year] [Name of Copyright Holder]. 

Should be `Copyright (C) 2021 Symbol Contributors`. CC @segfaultxavi, @AnthonyLaw and @fboucquez as this might be needed later on.